app/views/sulten/reservations/archive.html.haml

```diff
@@ -24,7 +24,7 @@
       %tr
         %td
           = link_to k.name, k
-          - unless k.allergies.empty?
+          - if k.allergies.present?
```

Comment:
`present` is better than `empty` because it won't raise a `NoMethodException` when `allergies` or some other value in the chain is `nil`. This also goes for `blank` below.

Reply:
Nice detail!